readme.txt

```diff
@@ -317,6 +317,7 @@ Please see the changelog for the complete list of changes in this release. Remem
 
 = [4.3.1] TBD =
 
+* Fix - Venue and Organizers urls are been generated correctly again, without the `-1` [68168]
```

Comment:
It's hard to translate the solution if you are not having a problem. I mean in a One-liner